en/editor/assets/index.md

```diff
@@ -28,6 +28,17 @@ The **Assets** panel can be divided into two main sections: **Toolbar** and **As
     - **Select All**: Ctrl/Cmd + A
     - **Rename**: Enter/F2
     - **Cancel input**: Esc
+    - **Copy**: Ctrl or Cmd + C
```

Comment:
These two parts are repeated:

![image](https://user-images.githubusercontent.com/19702899/116195972-79af5000-a765-11eb-91a1-4beecb0e69e3.png)

Reply:
that's weird, I'm not sure how that happened. fixed.